crates/biome_js_analyze/src/aria_analyzers/nursery/no_aria_hidden_on_focusable.rs

```diff
@@ -0,0 +1,123 @@
+use crate::{aria_services::Aria, JsRuleAction};
+use biome_analyze::{
+    context::RuleContext, declare_rule, ActionCategory, FixKind, Rule, RuleDiagnostic,
+};
+use biome_console::markup;
+use biome_diagnostics::Applicability;
+use biome_js_syntax::jsx_ext::AnyJsxElement;
+use biome_rowan::{AstNode, BatchMutationExt};
+
+declare_rule! {
+    /// Enforce that aria-hidden="true" is not set on focusable elements.
+    ///
```

Comment:
We should add some description of the rule. It is reason of existence.

Maybe something like:

> `aria-hidden="true"` can be used to hide purely decorative content from screen reader users.
> A focusable element with `aria-hidden="true"` can be reached by keyboard.
> This can lead to confusion or unexpected behavior for screen reader users.